openapi.yaml

```diff
@@ -8995,6 +8995,34 @@ paths:
           $ref: '#/components/responses/403'
         '404':
           $ref: '#/components/responses/404'
+  '/environment/{environmentId}/checkDockerfile':
+    post:
+      summary: Check dockerfile configuration is correct
+      operationId: checkDockerfile
+      parameters:
+        - $ref: '#/components/parameters/environmentId'
+      tags:
+        - environment
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DockerfileCheckRequest'
+      responses:
+        '201':
```

Comment:
I don't get why you're using http code 201 here 🤔

Reply:
copy/pasted from above. Changing it